rest/routing.go

```diff
@@ -357,6 +363,14 @@ func CreateMetricRouter(sc *ServerContext) *mux.Router {
 	return r
 }
 
+func CreateDiagnosticRouter(sc *ServerContext) *mux.Router {
+	r := mux.NewRouter()
+	r.StrictSlash(true)
+	r.Handle("/_ping", makeSilentHandler(sc, publicPrivs, nil, nil, (*handler).handlePing)).Methods("GET", "HEAD")
```

Comment:
I'd like to see a test that makes sure `_ping` works for all cases - there are two types of ways of testing this:

Add diagnostic port here https://github.com/couchbase/sync_gateway/blob/444f9b15fd6129f42be0dc45dc20959a63bc7d44/rest/api_test.go#L2814

This will test the router privs, but not whether the port is in use.